software/samba/en.md

```diff
@@ -1,40 +1,45 @@
 +++
 title = "Samba File Sharing"
-lastmod = "2018-05-19T15:07:07+03:00"
+lastmod = "2018-07-4T16:09:00+02:00"
 +++
 # Samba file sharing
 
-Samba is the standard Windows interoperability suite of programs for Linux and Unix. Among other things, it allows Linux and Windows users to share files via the Common Internet File System (CIFS) protocol.
+Samba is the standard Windows interoperability suite of programs for Linux and Unix. Among other things, it allows Linux, macOS and Windows users to share files via the Server Message Block (SMB) protocol.
 
 ## Samba on Solus
 
 To enable convenient file-sharing on Solus, we maintain a Solus-specific Samba configuration that out-of-the-box:
 
 - Supports Samba usershare functionality
 - Enables sharing of *$HOME* folders (manual user account activation needed)
-- Is set up as a standalone server which participates in the default MS Windows workgroup named WORKGROUP
+- Is set up as a standalone server using the SMB2.10 protocol and up (>= Win 7)
+- Advertises itself via *Avahi* aka *Apple Bonjour* / *mDNS* / *zeroconf*
 - Disables sharing of printers via Samba (use IPP via CUPS instead)
 - Allows access only from RFC 1918 private network addresses
 
+As of Samba 4.7.x, Solus disables the old, deprecated and insecure original SMB1/CIFS protocol by default.
+
+For more information on this choice, please visit http://aka.ms/stopusingsmb1
```

Comment:
Would prefer it be: `please read [this post regarding SMB1](https://blogs.technet.microsoft.com/filecab/2016/09/16/stop-using-smb1/).`